tex/CT4P.tex

```diff
@@ -1023,7 +1024,7 @@ \subsection{Initial Objects}
 However, in more complicated categories---that is, in categories where the objects and morphisms are complicated---an initial object can be very interesting; see, for instance, \cref{sec:initial-algs}.
 
 \begin{exer}
-  Does the category $\bullet$ have an initial object?
+  Does the category $\bullet$ (the terminal categroy defined in \cref{exa:graph_terminalcat}) have an initial object?
```

Comment:
```suggestion
  Does the category $\bullet$ (the terminal category defined in \cref{exa:graph_terminalcat}) have an initial object?
```

Reply:
Lol. I spent too long looking at these notes today. Sorry for that.